widgets/src/custom-elements/customEventsUtils.js

```diff
@@ -12,10 +12,21 @@ export const addCustomEventListener = (eventType, eventHandler) => {
   };
 };
 
+export const subscribeToWidgetEvents = (widgetEvents, eventHandler) => {
+  return widgetEvents.map(eventType => {
+    return addCustomEventListener(eventType, eventHandler);
+  });
+};
+
 const TASK_DETAILS_PREFIX = 'task.details';
 export const TD_ON_PRESS_PREVIOUS = `${TASK_DETAILS_PREFIX}.onPressPrevious`;
 export const TD_ON_PRESS_NEXT = `${TASK_DETAILS_PREFIX}.onPressNext`;
 export const TD_ON_ERROR = `${TASK_DETAILS_PREFIX}.onError`;
 
 const TASK_LIST_PREFIX = 'task.list';
 export const TL_ON_SELECT_TASK = `${TASK_LIST_PREFIX}.onSelectTask`;
+
+const PDA_CONFIG_PREFIX = 'pda.config';
+export const PDA_CONFIG_ON_UPDATE = `${PDA_CONFIG_PREFIX}.onUpdate`;
+
+export const BADGE_CHIP_INPUT_EVENTS = [PDA_CONFIG_ON_UPDATE];
```

Comment:
why do we need events for a badge?

Reply:
The Badge config should be saved in a upper level so it can be shared across the solution. As its setup relies on end user defined values, they cannot be on the level of component (each use would require a mapping of values/colors). @werbth and I have discussed some ways of how we will be able to pass the values, but if that will need a fetch (for example for global PDA configs), we have to make sure that badges are updated after the values are fetched.